### PR TITLE
Port log panel content to MUI (except List component)

### PR DESCRIPTION
**Pull request by foxymiles**
**User-Facing Changes**
None

**Description**
Ports the log panel content to MUI.

Partially addresses: #4079

### CODE REVIEW

**Comment by foxymiles** [comment]
We should look for a virtualized list component as part of porting this.

File: packages/studio-base/src/panels/Log/index.tsx [hunk 1]
```diff
@@ -194,40 +210,19 @@ const LogPanel = React.memo(({ config, saveConfig }: Props) => {
...
-              const normalizedLog = normalizedLogMessage(datatype, item["message"]);
-              return (
-                <LogMessage
-                  value={normalizedLog}
-                  timestampFormat={timeFormat}
-                  timeZone={timeZone}
-                />
-              );
-            }}
-          />
```
Comment (2metres) [inline packages/studio-base/src/panels/Log/index.tsx:221]: Should we switch over to use the List component from "react-virturalized"?

Comment (foxymiles) [reply packages/studio-base/src/panels/Log/index.tsx:221]: I've tried a couple of different libraries to replace the fluent list but so far haven't found one that performs as well.

**Comment by 2metres** [comment]
what about https://virtuoso.dev ?

**Comment by foxymiles** [comment]
> what about https://virtuoso.dev ?

Tried this and didn't get great results either.

**Comment by foxymiles** [comment]
Here's another one we could try https://tanstack.com/virtual/v3

**Review by 2metres — APPROVED** [review]

**Pull request merged**